accounts/forms.py

```
@@ -2,6 +2,7 @@
 from django.contrib.auth.models import User
 from django.contrib.auth.forms import UserCreationForm
 from django.core.exceptions import ValidationError
+from .models import UserBillingInfo
 
 class UserLoginForm(forms.Form):
     """Form to log users in"""
```

on line 31, you're excluding user records with the same username. that's not needed since we want to raise an error even if there's a record with the email and the username